elasticsearch/values.yaml

```diff
@@ -114,6 +114,7 @@ podSecurityPolicy:
       - secret
       - configMap
       - persistentVolumeClaim
+      - projected
```

Comment:
While `projected` doesn't seem required in my tests with GKE 1.16 it is mentionned as part of the recommended minimum set of allowed volumes for new PSPs in https://kubernetes.io/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems.

Note that we could also add `downwardAPI` and `emptyDir` which are also part of the recommendations (same for the other charts)